Auth0/JWTAlgorithm.swift

```diff
@@ -29,12 +29,10 @@ import Auth0ObjectiveC
 
 enum JWTAlgorithm: String {
     case rs256 = "RS256"
-    case hs256 = "HS256"
 
     var shouldVerify: Bool {
         switch self {
         case .rs256: return true
-        case .hs256: return false
         }
     }
```

Comment:
Does it still make sense for this to be in a `switch` given it only has one option, or can it be simplified?

Reply:
It does, since we can have more algorithms in the future.